tests/fixtures/toolsObjectData.js

```diff
@@ -0,0 +1,275 @@
+const mockData = {
+  items: [
+    {
+      name: '.asyncapi-tool-example',
+      url: 'https://api.github.com/repositories/351453552/contents/.asyncapi-tool?ref=61855e7365a881e98c2fe667a658a0005753d873',
+      repository: {
+        full_name: 'asyncapi/example-repo',
+        html_url: 'https://github.com/asyncapi/example-repo',
+        description: 'Example repository',
+        owner: {
+          login: 'asyncapi'
+        }
+      },
+      path: '.asyncapi-tool'
+    }
+  ]
+};
+
+const mockToolFileContent = `
+  title: Example Tool
+  description: This is an example tool.
+  links:
+    repoUrl: https://github.com/asyncapi/example-repo
+  filters:
+    categories:
+      - Category1
+    hasCommercial: true
+  `;
+
+const toolFileT1 = {
+  title: 'Example Tool',
+  description: 'This is an example tool.',
+  links: {
+    repoUrl: 'https://github.com/asyncapi/example-repo'
+  },
+  filters: {
+    categories: ['Category1'],
+    hasCommercial: true
+  }
+};
+
+const expectedObjectT1 = {
+  title: 'Example Tool',
+  description: 'This is an example tool.',
+  links: {
+    repoUrl: 'https://github.com/asyncapi/example-repo'
+  },
+  filters: {
+    categories: ['Category1'],
+    hasCommercial: true,
+    isAsyncAPIOwner: true
+  }
+};
+
+const repositoryUrl = 'https://github.com/asyncapi/example-repo';
+const repoDescription = 'Example repository';
+const isAsyncAPIOwner = true;
+
+const toolFileT2 = {
+  title: 'Example Tool',
+  links: {
+    repoUrl: 'https://github.com/asyncapi/example-repo'
+  },
+  filters: {
+    categories: ['Category1']
```

Comment:
I think you should use a real category name here instead of taking any arbitrary value.

Reply:
@vishvamsinh28 Kindly address this.